src/components/NewLocationPage/CompareLocationTabs/CompareLocationTabs.style.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
+
+export const ButtonGroup = styled(ToggleButtonGroup)`
+  background-color: ${COLOR_MAP.GREY_1};
+  border: 1px solid ${COLOR_MAP.GREY_2};
+  border-radius: 4px;
+`;
+
+export const Button = styled(ToggleButton)`
+  padding: 0.63rem;
+  border: none;
+  .MuiToggleButton-label {
+    color: ${COLOR_MAP.GREY_3};
+    text-transform: none;
+    line-height: 1rem;
```

Comment:
lets standardize to `line-height: 1.4`. when we don't specify a unit (ie, `1.4` instead of `1rem`), the # is multiplied by the element's font-size, so it stays relative/scales